src/main/java/org/testmonkeys/jentitytest/matchers/EntityMatcher.java

```diff
@@ -48,4 +50,15 @@ public void describeTo(Description description) {
         description.appendText("Entities have same values in properties");
     }
 
+    private StringBuilder showControlChars(Object inputObj) {
+        StringBuilder res = new StringBuilder(inputObj.toString());
+        for (int i = 0; i < controlChars.length; i++) {
+            int index = res.indexOf(String.valueOf(controlChars[i]));
```

Comment:
with current implementation only first occurrence will be escaped, I would suggest making into a loop while index not -1, and continuing from last index